### DISCUSSION TITLE
Collector forwarding data

### DISCUSSION THREAD

**Discussion by 4quarks** [dmachard/DNS-collector#796] [Q&A]
Hello everyone,

I’d like to use the collector as a "middleman" to simply forward data. In my DNS configuration, I have something similar to [1], but I’m curious if it’s possible to configure the collector to only receive data and send it to another instance without sniffing the data itself.

Thanks in advance for any insights!

[1]
multiplexer:
  collectors:
    - name: sniff
      afpacket-sniffer:
        device: eth0 
        port: 53 
        chan-buffer-size: 65535
      transforms:
        filtering:
          log-queries: false
          log-replies: true

  loggers:
    - name: remote_instance
      dnstapclient:
        remote-address: server_ip 
        remote-port: server_port 
        connect-timeout: 5
        retry-interval: 10
        flush-interval: 10
        tls-support: false
        tls-insecure: false
        server-id: "ID"
        buffer-size: 100
        chan-buffer-size: 65535

  routes:
    - from: [ sniff ]
      to: [ remote_instance ]

**Comment by dmachard** [comment]
You mean, forward DNS packets without decoding or inspecting them?"

**Reply by 4quarks** [reply]
Exactly, the final receiver should get the same information as a direct receiver ([1]) would without any transformation.

**Reply by dmachard** [reply]
It's available but only with the [DNStap proxifier](https://github.com/dmachard/go-dnscollector/blob/main/docs/collectors/collector_dnstap.md#dns-tap-proxifier)
